### ISSUE TITLE
[stable/postgres] Kubernetes v1.16 support

### ISSUE THREAD

**Issue by jaredallard** [helm/charts#17795]
**Describe the bug**

Attempted to install a chart using postgres:

```
Error: apiVersion "apps/v1beta2" in <parentChart>/charts/postgres/templates/statefulset.yaml is not available
```

**Version of Helm and Kubernetes**:

```
$ helm3 version
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}
$ kubectl version
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.1", GitCommit:"d647ddbd755faf07169599a625faf302ffc34458", GitTreeState:"clean", BuildDate:"2019-10-02T16:51:36Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:

stable/postgres 5.3.13

**What happened**:

See bug

**What you expected to happen**:

Postgres was created successfully

**How to reproduce it** (as minimally and precisely as possible):

Install `stable/postgres` on a Kubernetes v1.16 cluster

**Anything else we need to know**:

Might want to make this configurable, or determine it based on kubernetes versions passed to helm. I can provide a PR for that behaviour if wanted.

Related to #17779

**Comment by EricLemieux** [comment]
I'll create a pr

**Comment by EricLemieux** [comment]
Actually it looks like this was fixed in `6.3.9`, #17281

**Comment by jaredallard** [comment]
Oops... My bad! Closing this then.
